chef/cookbooks/neutron/recipes/post_install_conf.rb

```diff
@@ -84,7 +84,11 @@ def mask_to_bits(mask)
     end
   when 'gre'
     fixed_network_type = "--provider:network_type gre --provider:segmentation_id 1"
-    floating_network_type = "--provider:network_type gre --provider:segmentation_id 2"
+    if node[:network][:networks][:nova_floating][:use_vlan]
```

Comment:
yeah, I was wondering about that- I'm not sure about the vxlan case (see below).